src/sound_netbsd.c

```diff
@@ -36,7 +36,7 @@ static int init(struct options *options)
 	chkparm1("buffer", bsize = strtoul(token, NULL, 0));
 	parm_end();
 
-	if ((audio_fd = open("/dev/sound", O_WRONLY)) == -1)
+	if ((audio_fd = open("/dev/audio", O_WRONLY)) == -1)
```

Comment:
Would it make sense to have a list of device paths like the OSS driver does, and fall back on `/dev/sound` if `/dev/audio` can't be found?